_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -1709,10 +1711,12 @@ mfxStatus VAAPIEncoder::CreateAccelerationService(MfxVideoParam const & par)
 #if defined(MFX_ENABLE_H264_VIDEO_FEI_ENCPAK) || defined(MFX_ENABLE_H264_VIDEO_FEI_PREENC)
     if ( m_isENCPAK )
     {
-        attrib[2].type = VAConfigAttribFEIFunctionType;
+        attrib[numAttrib].type = VAConfigAttribFEIFunctionType;
```

Comment:
I don't like this hack with ```numAttrib``` variable. Better to use ```push_back``` and don't have explicit index manipulation

Reply:
Done.